content/docs/news/vulnerability_bulletins/keptn-sec-2020-003/index.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Keptn-Vulnerability-2020-003
+description: RBAC cluster-admin role given to Keptn services by default
```

Comment:
Copy-pasta, as well as file name with `2020` in it

Reply:
thx, renamed it accordingtly.